jvm/src/streamparse/commands/kill_topology.clj

```diff
@@ -30,7 +30,7 @@
   (let [[opts args banner]
          (cli args
               ["-h" "--help" "Show this help screen." :flag true :default false]
-              ["-w" "--wait" "Amount of time to wait for topology to gracefully stop." :default 10 :parse-fn #(Integer/parseInt %)]
+              ["-w" "--wait" "Amount of time to wait for topology to gracefully stop." :parse-fn #(Integer/parseInt %)]
```

Comment:
I didn't. That was an assumption from how kill works on the Storm UI. 

Can you think of a better way of testing beyond refreshing the UI to see when the topo finally dies?